CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# dbt_jira v0.11.1
```

Comment:
```suggestion
# dbt_jira v0.12.0
```
Note to make this a breaking change and add a note about probably needing a full refresh.